tests/contrib/aiohttp/test_middleware.py

```diff
@@ -114,6 +114,40 @@ def test_404_handler(self):
         assert 'GET' == span.get_tag('http.method')
         assert '404' == span.get_tag('http.status_code')
 
+    @unittest_run_loop
+    @asyncio.coroutine
+    def test_server_error(self):
+        """
+        When a server error occurs (uncaught exception)
+            The span should be flagged as an error
+        """
+        request = yield from self.client.request('GET', '/uncaught_server_error')
+        assert request.status == 500
+        traces = self.tracer.writer.pop_traces()
+        assert len(traces) == 1
+        assert len(traces[0]) == 1
+        span = traces[0][0]
+        assert span.get_tag('http.method') == 'GET'
+        assert span.get_tag('http.status_code') == '500'
+        assert span.error == 1
```

Comment:
any way for us to get the actual exception object and attach to the span? (do we do that already?)

Reply:
(it seems like we do, so can ignore this)